From 5c5b9719e8c0bf627b16a39c927cb2f3685357b2 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:23:20 -0400
Subject: [redhat] kabi: Adding symbol find_or_create_page (mm/filemap.c)

Message-id: <1b4d1673b97cfab8989432388609d0b4f253393a.1410510773.git.poros@redhat.com>
Patchwork-id: 96091
O-Subject: [PATCH RHEL7.1 080/233] kabi: Adding symbol find_or_create_page (mm/filemap.c)
Bugzilla: 1071098
RH-Acked-by: Larry Woodman <lwoodman@redhat.com>

Bugzilla: 1071098
    https://bugzilla.redhat.com/show_bug.cgi?id=1071098

    Description:
    ============
    Locate or add a pagecache page.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/find_or_create_page
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/find_or_create_page
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/find_or_create_page

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/find_or_create_page b/redhat/kabi/kabi-module/kabi_ppc64/find_or_create_page
new file mode 100644
index 0000000..fff58b2
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/find_or_create_page
@@ -0,0 +1,2 @@
+#1-
+0x0892891a find_or_create_page vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/find_or_create_page b/redhat/kabi/kabi-module/kabi_s390x/find_or_create_page
new file mode 100644
index 0000000..36acd87
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/find_or_create_page
@@ -0,0 +1,2 @@
+#1-
+0x575a8217 find_or_create_page vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/find_or_create_page b/redhat/kabi/kabi-module/kabi_x86_64/find_or_create_page
new file mode 100644
index 0000000..850890c
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/find_or_create_page
@@ -0,0 +1,2 @@
+#1-
+0x0cd25c12 find_or_create_page vmlinux EXPORT_SYMBOL
-- 
1.7.1